### DISCUSSION TITLE
Getting Files Into BookPlayer Using iTunes File Sharing

### DISCUSSION THREAD

**Discussion by Dahveed7** [TortugaPower/BookPlayer#578] [Q&A] [answered]
I'm trying to use iTunes file sharing to get my files into BookPlayer. I could have sworn that is how I did it before but it doesn't seem to work now. The audiobook folder SSNL gets copied somewhere but it doesn't show up in BookPlayer on my iPhone. Am I missing something??

![Screen Shot 2021-04-08 at 9 43 02 PM](https://user-images.githubusercontent.com/82148605/114006313-73d7f480-98b4-11eb-8f80-d4df7600825f.jpg)

**Comment by GianniCarlo** [comment, marked as answer]
Hi @Dahveed7 ! yes, some versions ago we disabled direct folder support (due to a bug because of how Airdrop receive files one at a time), we're near bringing that back, but in the meantime, you can zip the folder, and transfer that zip file instead, then BookPlayer will process that

**Comment by Dahveed7** [comment]
Thanks! That worked. :-)
